src/content/components/ReleaseReport/ReleaseReport.js

```diff
@@ -54,47 +40,187 @@ export class ReleaseReport extends React.PureComponent {
+                );
+                return (
+                  <div key={meta.id} className={styles.feature}>
+                    <h3 className={styles.h3}>
+                      {meta.displayName} ({completionPercentage}% complete)
+                    </h3>
+                    <p className={styles.featureSummary}>{meta.description}</p>
+                    <CompletionBar
+                      bugs={bugs}
+                      startDate="2018-03-01"
+                      endDate="2018-04-29"
+                    />
+                    <ul className={styles.bugList}>
+                      {bugs.map(bug => (
+                        <li
+                          key={bug.id}
+                          className={isBugResolved(bug) ? styles.resolved : ""}>
+                          <a href={OPEN_BUG_URL + bug.id}>{bug.summary}</a>
+                        </li>
+                      ))}
+                    </ul>
+                  </div>
+                );
+              })
+          ) : (
+            <Loader />
+          )}
+        </div>
+      </div>
+    );
+  }
+}
 
-        {this.state.loaded ? (
-          this.props.metas
-            .filter(meta => meta.priority === "P1")
-            .map(meta => {
-              const bugs = this.state.bugs.filter(b =>
-                b.blocks.includes(meta.id)
-              );
-              if (!bugs.length) {
-                return null;
-              }
-              const completionPercentage = Math.round(
-                (bugs.filter(isBugResolved).length / bugs.length) * 100
-              );
-              return (
-                <div key={meta.id} className={styles.feature}>
-                  <h3 className={styles.h3}>
-                    {meta.displayName} ({completionPercentage}% complete)
-                  </h3>
-                  <p className={styles.featureSummary}>{meta.description}</p>
-                  <CompletionBar
-                    bugs={bugs}
-                    startDate="2018-03-01"
-                    endDate="2018-04-29"
-                  />
-                  <ul className={styles.bugList}>
-                    {bugs.map(bug => (
-                      <li
-                        key={bug.id}
-                        className={isBugResolved(bug) ? styles.resolved : ""}>
-                        <a href={OPEN_BUG_URL + bug.id}>{bug.summary}</a>
-                      </li>
-                    ))}
-                  </ul>
-                </div>
-              );
-            })
-        ) : (
-          <Loader />
-        )}
+class ReleaseDatesTab extends React.PureComponent {
+  getStartDate(milestone) {
+    let iterationDate = ITERATION_LOOKUP.byVersionString[milestone];
+    let milestoneDate = new Date(iterationDate.startDate);
+
+    return milestoneDate;
+  }
+
+  getMilestoneDate(milestone) {
+    const iteration = getIteration();
+    const release = iteration.number.split(".")[0];
```

Comment:
right now `release` is a `string` and `nextRelease` is a `number`, it probably makes more sense for both of them to just be `number`s just to be consistent

Reply:
`release` is used as a string in the HTML a lot - I pretty much did parseInt() so I could add +1. If I convert things to Luxon I think that won't be necessary any more